pkg/adapter/adapter.go

```diff
@@ -65,6 +67,19 @@ func NewLocalAdapter(cfg *config.Config) (*LocalAdapter, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "create content provider")
 	}
+	c := cron.New()
+	threshold, err := humanize.ParseBytes(cfg.Provider.GCPolicy.Threshold)
+	if err != nil {
+		return nil, err
+	}
+	if _, err := c.AddFunc("@hourly", func() {
```

Comment:
Is there any reason to introduce a package to support cron jobs?

Reply:
> Is there any reason to use this package to support cron jobs?

Actually, just consider the convenience and possible new requirements in the future, have any better suggestions?